pallets/token-manager/src/test_avt_tokens.rs

```diff
@@ -341,3 +396,47 @@ fn avn_test_avt_token_total_lowered_amount_greater_than_balance_max_value_ok() {
             })));
     });
 }
+
+#[test]
+fn avt_lower_claimed_succesfully() {
+    let mut ext = ExtBuilder::build_default()
+        .with_genesis_config()
+        .with_balances()
+        .as_externality();
+
+    ext.execute_with(|| {
+        let mock_data = MockData::setup(AMOUNT_123_TOKEN, USE_RECEIVER_WITH_0_AMOUNT);
+        let mock_event = &mock_data.lower_claimed_event;
+        insert_to_mock_processed_events(&mock_event.event_id);
+
+        let lower_id = 0;
+
+        perform_lower_setup(lower_id);
+
+        assert_ok!(TokenManager::on_event_processed(&mock_event));
+
+        assert!(System::events().iter().any(|a| a.event ==
```

Comment:
might as well add that the lower is not present in the storage item ReadyToClaim